blackbox/docs/general/builtins/scalar.rst

```diff
@@ -2065,7 +2065,8 @@ which in CrateDB will always be ``crate``::
 ``current_setting(text [,boolean])``
 --------------------------------------
```

Comment:
this was me missing it, but could you please fix the length (sorry, just noticed it)